tests/cases/events_test.py

```diff
@@ -136,3 +139,14 @@ def callback(event):
             self.assertEqual(events.daemon.eventQueue.qsize(), 0)
             self.assertEqual(self.ctr, 3)
             self.assertEqual(self.responses, ['foo'])
+
+    def testDisableDaemon(self):
+        self.assertTrue(hasattr(events.daemon, 'start'))
+
+        # Disable the event daemon and reload the module
+        config.getConfig()['server']['disable_event_daemon'] = True
+        del sys.modules['girder.events']
+        eventsModule = importlib.import_module('girder.events')
+
+        self.assertFalse(hasattr(eventsModule.daemon, 'start'))
+        self.assertTrue(callable(eventsModule.daemon.trigger))
```

Comment:
is it possible to run `testAsyncEvents` with `disable_event_daemon` set?  it would be nice to test the behavior in some way in the case of no async thread, as clients can still call daemon methods.